src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -78,7 +56,7 @@ protected function buildEntityQuery($match = NULL, $match_operator = 'CONTAINS')
 
     $ids = [];
 
-    if ($this->configuration['handler_settings']['field_mode'] == 'admin') {
+    if ($this->isGroupAdmin()) {
```

Comment:
OK, Sure. I didn't mean remove the logic, just the fact that it is a configuration setting. That looks like it doesn't make too much sense?